sdk/communication/communication-sms/test/utils/recordedClient.ts

```diff
@@ -18,6 +18,11 @@ export const recorderConfiguration: RecorderEnvironmentSetup = {
   queryParametersToSkip: []
 };
 
+export function communicationEndpoint(): string {
```

Comment:
Since you aren't doing any checks on the string; and you only ever use this once; you can just do this parsing inline instead of exporting a function.